src/constants.js

```diff
@@ -57,4 +57,4 @@ export const COURSE_BLOCK_NAMES = /** @type {const} */ ({
   component: { id: 'component', name: 'Component' },
 });
 
-export const UPLOAD_FILE_MAX_SIZE = 100 * 1024 * 1024; // 100mb
+export const UPLOAD_FILE_MAX_SIZE = 20 * 1000 * 1000; // 20mb
```

Comment:
When I have a file with such size as shown on the picture it will `20312736`.
<img width="175" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/36768631/6d4e5c09-f43b-4320-890c-7b3d37265f24">
In that time if I set `20 * 1024 * 1024` it results in `20971520`. This means that `20312736` is valid size. But when I upload to the server I receive `413` error that means the file is too large.

The solution is to set `20 * 1000 * 1000` that works good for both backend and frontend.